widgetschema.json

```diff
@@ -162,6 +195,7 @@
                         "choices",
                         "inputGroup",
                         "date",
+                        "multiple",
```

Comment:
I see we already have a `isMultiple` flag.  How does that relate to this new `multiple` type?  Or how is `multiple` different from `inputGroup` with `isMultiple = true`?

Reply:
I guess I do like that idea.  I'll definitely say that having a `multiple` type and a `isMultiple` flag that are completely unrelated seems confusing.